pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/GrpcReceivingMailbox.java

```diff
@@ -94,6 +91,10 @@ public boolean isClosed() {
     return isInitialized() && _contentStreamObserver.isCompleted();
   }
 
+  @Override
+  public void cancel(Throwable e) {
```

Comment:
shouldn't we clean up the `_contentStreamObserver` in this method?

Reply:
This PR only introduces API. no actual implementation is done.